### PR TITLE
clean code style issues

### PR DESCRIPTION
**Pull request by nlu90**
### Motivation

There're many code style issues as indicated by codacy.

### Modifications

1. throw NPE
2. logic inversion
3. unnecessary case change
4. tailing semicolon

### Verifying this change

- [x] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [X] `no-need-doc`  code-style changes
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: pulsar-broker/src/test/java/org/apache/pulsar/client/api/InterceptorsTest.java [hunk 1]
```diff
@@ -151,12 +151,12 @@ public void close() {
 
             @Override
             public Message<String> beforeSend(Producer<String> producer, Message<String> message) {
-                throw new NullPointerException();
+                throw new IllegalArgumentException();
```
Comment (Jason918) [inline pulsar-broker/src/test/java/org/apache/pulsar/client/api/InterceptorsTest.java:154]: Maybe NotImplementException is better?

File: pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/PrimitiveSchemaTest.java [hunk 2]
```diff
@@ -124,7 +124,7 @@ public void allSchemasShouldSupportNull(Map<Schema, List<Object>> testData) {
                 assertNull(((AbstractSchema) schema).decode(byteBuf),
                     "Should support null in " + schema.getSchemaInfo().getName() + " deserialization");
             } catch (NullPointerException npe) {
-                throw new NullPointerException("NPE when using schema " + schema + " : " + npe.getMessage());
+                throw new IllegalArgumentException("NPE when using schema " + schema + " : " + npe.getMessage(), npe);
```
Comment (Jason918) [inline pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/PrimitiveSchemaTest.java:127]: Why change NullPointerException to IllegalArgumentException here?

Comment (nlu90) [reply pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/PrimitiveSchemaTest.java:127]: `Avoid throwing NullPointerExceptions manually.` is the guide line I saw from codacy.

**Review by hangc0276 — APPROVED** [review]

File: pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/protobufnative/TestMsg.java [hunk 3]
```diff
@@ -190,7 +190,7 @@ private SubMessage(
         throws com.google.protobuf.InvalidProtocolBufferException {
       this();
       if (extensionRegistry == null) {
-        throw new java.lang.NullPointerException();
+        throw new java.lang.IllegalArgumentException();
```
Comment (codelipenghui) [inline pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/protobufnative/TestMsg.java:193]: Looks like the TestMsg is generated by protobuf, we can skip this class.

File: pulsar-broker/src/test/java/org/apache/pulsar/client/api/InterceptorsTest.java (hunk 1 above)
Comment (codelipenghui) [reply pulsar-broker/src/test/java/org/apache/pulsar/client/api/InterceptorsTest.java:154]: This test is testing the interceptor throws exceptions, so it's ok for any runtime exceptions.

File: pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/protobufnative/TestMsg.java (hunk 3 above)
Comment (nlu90) [reply pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/protobufnative/TestMsg.java:193]: 👍🏼

Comment (nlu90) [reply pulsar-sql/presto-pulsar/src/test/java/org/apache/pulsar/sql/presto/decoder/protobufnative/TestMsg.java]: @codelipenghui I've reverted the changes inside `TestMsg` file

**Comment by nlu90** [comment]
@codelipenghui let me know if there's any change need to be done before we merge this PR.

**Review by freeznet — APPROVED** [review]

**Pull request merged**